pkg/client/clientset/versioned/clientset.go

```diff
@@ -39,31 +39,36 @@ import (
 
 type Interface interface {
 	Discovery() discovery.DiscoveryInterface
+	EventingV1() eventingv1.EventingV1Interface
 	EventingV1alpha1() eventingv1alpha1.EventingV1alpha1Interface
 	EventingV1beta1() eventingv1beta1.EventingV1beta1Interface
 	EventingV1beta2() eventingv1beta2.EventingV1beta2Interface
 	EventingV1beta3() eventingv1beta3.EventingV1beta3Interface
-	EventingV1() eventingv1.EventingV1Interface
 	FlowsV1() flowsv1.FlowsV1Interface
 	MessagingV1() messagingv1.MessagingV1Interface
 	SinksV1alpha1() sinksv1alpha1.SinksV1alpha1Interface
-	SourcesV1beta2() sourcesv1beta2.SourcesV1beta2Interface
 	SourcesV1() sourcesv1.SourcesV1Interface
+	SourcesV1beta2() sourcesv1beta2.SourcesV1beta2Interface
 }
 
 // Clientset contains the clients for groups.
 type Clientset struct {
 	*discovery.DiscoveryClient
+	eventingV1       *eventingv1.EventingV1Client
```

Comment:
I guessing the ordering changes are just the drift from Kube 0.29 to 0.30?

Reply:
yes